app/views/administrate/application/_sidebar.html.erb

```diff
@@ -11,7 +11,7 @@ as defined by the DashboardManifest.
   <% DashboardManifest::DASHBOARDS.each do |resource| %>
     <li>
       <%= link_to(
-        resource.to_s.titleize,
+        resource.to_s.classify.constantize.model_name.human(default: resource.to_s.titleize),
```

Comment:
No. The user decides how to display the model name through a file locale. In default pluralization.